pipeline-modules/app-service/gcp/docker-build.yaml

```diff
@@ -0,0 +1,38 @@
+provisioner: gcp
+name: docker-build
+version: 1
+revision: 1
+displayName: Docker Build
+description: This plugin only builds the Docker image, useful for running checks
+target: ""
+category: containerise
+keywords:
+  - Docker
+  - Build
+author: CloudCover
+meta: {}
+inputs:
+  properties:
+    docker_build_args:
+      title: Docker Build Arguments
+      description: Arguments for the docker build command
+      type: array
+      default: []
+      items:
+        type: string
+    working_dir:
+      title: Working directory
+      description: The current working directory to execute command
+      type: string
+      default: repo
+template: |
+  steps:
+  - id: 'Docker Build'
+    name: 'gcr.io/cloud-builders/docker'
+    dir: {{ working_dir }}
+    args:
+    - '-c'
```

Comment:
I believe this -c was leftover from when the entrypoint was being overridden to "bash"...when -c is passed to "docker" it is to set the directory for the docker context. 
I'm confused how this works because the way i read it you will end up with this command running (if docker_build_args is "build"):
$ docker -c t temp build
I don't believe that command will parse? (I get "unable to resolve docker endpoint: context "t" does not exist")

Reply:
Fixed